addOns/custompayloads/src/main/java/org/zaproxy/zap/extension/custompayloads/CustomPayloadsMultipleOptionsTablePanel.java

```diff
@@ -64,6 +69,7 @@ public CustomPayloadsMultipleOptionsTablePanel(
         addMissingDefaultsButton();
         addResetButton();
         addResetIdButton();
+        addButton();
```

Comment:
Should have a button spacer to separate from the other unrelated functionalities.